src/w3d/renderer/render2dsentence.cpp

```diff
@@ -1005,7 +1012,8 @@ Vector2 Render2DSentenceClass::Build_Sentence_Not_Centered(const unichar_t *text
                                 ++tmp_text;
                             }
 
-                            spacing += m_font->Get_Char_Spacing(*tmp_text++);
+                            spacing += m_font->Get_Char_Spacing(*tmp_text);
+                            ++tmp_text;
```

Comment:
Why this change its identical

Reply:
It's easier on the eye in this case and compares with the decompile better.